cmd/testrunner.go

```diff
@@ -1,21 +1,60 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/packages"
+	"github.com/elastic/elastic-package/internal/testrunner/system"
 )
 
 func setupTestCommand() *cobra.Command {
-	return &cobra.Command{
+	testSystemCmd := &cobra.Command{
+		Use:   "system",
+		Short: "Run system tests",
+		Long:  "Run system tests for a package.",
+		RunE:  testSystemCommandAction,
+	}
+
+	cmd := &cobra.Command{
 		Use:   "test",
 		Short: "Run test suite for the package",
 		Long:  "Use test runners to verify if the package collects logs and metrics properly.",
 		RunE:  testCommandAction,
 	}
+	cmd.PersistentFlags().BoolP("ignoreMissing", "i", false, "ignore missing tests")
```

Comment:
Hm... I wonder if the command should ignore the fact that there are no tests by default. Similarly to:

```bash
➜  elastic-package git:(promote-subcommand) go test
?   	github.com/elastic/elastic-package	[no test files]
➜  elastic-package git:(promote-subcommand) echo $?
0
```

Reply:
Yeah, I went back and forth on this. I'm okay ignoring missing tests as the default behavior. To reflect this better, should I change the flag to be something like `--failOnMissingTests/-m`, defaulting to `false`?

Reply:
Works for me!

It might be personal preference, but maybe let's stick to dashes: `--fail-on-missing`.

Reply:
Hmmm, I prefer the kebab case too but I'm not sure this is supported. Will try though. Are you also suggesting we remove the short option (`-m`) or just that you prefer the kebab case over the camel case for the long option?

Reply:
Sorry, I missed it. The short option can stay there.

Reply:
Changed `--failOnMissing` to `--fail-on-missing` in 72a3ded.